src/components/Payment/Payment.js

```diff
@@ -188,8 +202,7 @@ class Payment extends Component {
             <MethodsWrapperStyled>
               {paymentMethods.map(
                 method =>
-                  method.methodName !== 'manual' &&
-                  availablePaymentMethods.includes(method.paymentGateway) && (
```

Comment:
we could remove `method.methodName !== 'manual'` and just keep the validation if padded payment methods are supported